packages/kit/src/components/AddressInfo/index.tsx

```
@@ -155,62 +155,63 @@ function AddressInfo(props: IProps) {
     return null;
   }
 
-  const AccountNameContainer = allowClickAccountNameSwitch
-    ? SwitchHomeAccountContainer
-    : Stack;
+  const renderWalletAccountName = () => {
+    if (!addressQueryResult?.walletAccountName) return null;
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)